news/12791.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Improve performance. Fix code to display installed packages at the end of
+ pip install, was O(n^2) to enumerate packages due to accidental loop in loop.
```

Comment:
```suggestion
Improve pip install performance. The installed packages printout is
now calculated in linear time instead of quadratic time.
```

I think this reads a bit better. Lemme know what you think.